src/components/NewCommentForm/NewCommentForm.jsx

```diff
@@ -1,39 +1,73 @@
-import React from 'react';
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+
+import { addComment, getPostComments } from '../../api/comments';
+
 import './NewCommentForm.scss';
 
-export const NewCommentForm = () => (
-  <form className="NewCommentForm">
-    <div className="form-field">
-      <input
-        type="text"
-        name="name"
-        placeholder="Your name"
-        className="NewCommentForm__input"
-      />
-    </div>
-
-    <div className="form-field">
-      <input
-        type="text"
-        name="email"
-        placeholder="Your email"
-        className="NewCommentForm__input"
-      />
-    </div>
-
-    <div className="form-field">
-      <textarea
-        name="body"
-        placeholder="Type comment here"
-        className="NewCommentForm__input"
-      />
-    </div>
-
-    <button
-      type="submit"
-      className="NewCommentForm__submit-button button"
-    >
-      Add a comment
-    </button>
-  </form>
-);
+export const NewCommentForm = ({ postId, setComments }) => {
+  const [name, setName] = useState('');
+  const [email, setEmail] = useState('');
+  const [body, setBody] = useState('');
+
+  const handleSubmit = (event) => {
+    event.preventDefault();
+    if (name && email && body) {
+      addComment(postId, name, email, body)
+        .then(() => getPostComments(postId)
+          .then(setComments));
+    }
+
```

Comment:
```suggestion
    addComment(postId, name, email, body)
      .then(() => getPostComments(postId)
        .then(setComments));
```
You can add the attribute ```required``` for everyone's input.